src/core/_buttons.css

```diff
@@ -19,6 +19,11 @@ input[type="button"] {
   border-radius: var(--border-radius);
   border: var(--button-border);
   cursor: pointer;
+  -webkit-transition: border 0.2s;
```

Comment:
The postCSS options can be configured here https://github.com/gld1982ltd/skeleton-postcss as we agreed to purge this repo of build libs. I am actually leaning toward postcss-cli as our runner allowing everything to be configured and ran from the same package.json.